src/main/java/org/twilmes/sql/gremlin/processor/GremlinCompiler.java

```diff
@@ -121,7 +121,7 @@ public String explain(String sql) {
             final FieldMapVisitor fieldMapper = new FieldMapVisitor();
             new RelWalker(root, fieldMapper);
             final TraversalVisitor traversalVisitor = new TraversalVisitor(graph.traversal(),
-                    scanMap, fieldMapper.getFieldMap());
+                    scanMap, fieldMapper.getFieldMap(), schemaConfig);
```

Comment:
When doing 3 tables or more table joins, I see parser gives path that are joining two tables that has no relationship described in SchemaConfig, why? I think it should be problem is Calcite, here I was thinking of providing schema to checking in TraversalVisitor, but was not able to find left and right gremlin table in any of these nodes, so this will be a TODO item to check why it join two tables that has no relationship defined.